concert/ext/tangoservers/qa.py

```diff
@@ -0,0 +1,318 @@
+"""
+qa.py
+-----
+Implements a device server to execute quality assurance routines during acquisition.
+"""
+from typing import List, AsyncIterator, Tuple
+import numpy as np
+from numpy.typing import ArrayLike
+import scipy.ndimage as snd
+import scipy.optimize as sop
+import skimage.filters as skf
+import skimage.measure as sms
+from skimage.measure._regionprops import RegionProperties
+from tango import DebugIt, DevState, CmdArgType, EventType
+from tango.server import attribute, command, AttrWriteType
+from concert.ext.tangoservers.base import TangoRemoteProcessing
+from concert.ext.ufo import FlatCorrect, GaussianFilter, MedianFilter
+
+
+class QualityAssurance(TangoRemoteProcessing):
```

Comment:
This name does not represent what it is doing, how about `RotationAxisEstimator`?